src/modules/auth/api.js

```diff
@@ -53,11 +53,17 @@ export class SeaCatAuthApi {
 
 
 	userinfo(access_token) {
-		let headers = {}
+		let headers = {};
+		// Obtain active tenant from URL params (if available)
+		const search = window.location.search;
+		const params = new URLSearchParams(search);
+		let active_tenant = params.get('tenant');
+		// Add access bearer token to the Authorization headers
 		if (access_token != null) {
 			headers.Authorization = 'Bearer ' + access_token;
 		}
-		return this.OidcAPI.get('/userinfo', {headers: headers});
+		// The userinfo API call will handle the cases where tenant is not available or undefined
+		return this.OidcAPI.get('/userinfo?tenant=' + active_tenant, {headers: headers});
```

Comment:
Still - I don't want to generare request with "?tenant=" for deployments where tenants are not used.

Reply:
Oh, sure, sorry, I misunderstood. https://github.com/TeskaLabs/asab-webui/pull/94/commits/bfc6e1a7617e5a1b848137b132dc29db0d50d0a1

I also changed the URL of Access control screen (it is the screen which is displayed in the header dropdown with the information about accesses) when the tenant is not present. https://github.com/TeskaLabs/asab-webui/pull/94/commits/20fcbf75bc4fa935a963d6a021489bb2f218a31d